chaos.proto

```diff
@@ -82,8 +82,34 @@ message Impact{
 
     //In POSIX time (i.e., number of seconds since January 1st 1970 00:00:00 UTC).
     optional uint64 notification_date = 9;
+
+    // Application pattern used while creating impact if any
+    repeated Pattern application_patterns = 10;
+}
+
+message WeekPattern {
+    optional bool monday = 1;
+    optional bool tuesday = 2;
+    optional bool wednesday = 3;
+    optional bool thursday = 4;
+    optional bool friday = 5;
+    optional bool saturday = 6;
+    optional bool sunday = 7;
 }
 
+message TimeSlot{
+	// Times in local for a timeslot
+	required uint32 begin = 1;
+	required uint32 end = 2;
+}
+
+message Pattern {
+	// Dates in local
+	required uint32 start_date = 1;
```

Comment:
:ok:  I don't like that part of the feed is in UTC (POSIX time) and part local, but it matches NTFS https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md#format-des-donn%C3%A9es